<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>453</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Escala Personalizada</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">
QCalendarWidget QWidget {
	 alternate-background-color: #B8E2FF;
}

/* style for top navigation area ###############################################*/ 

#qt_calendar_navigationbar {
    background-color: #fff;
	border: 2px solid  #B8E2FF;
	border-bottom: 0px;
	border-top-left-radius: 5px;
	border-top-right-radius: 5px;
}

/* style for month change buttons ############################################ */

#qt_calendar_prevmonth, 
#qt_calendar_nextmonth {
	/* border delete */
    border: none;  
    /* delete default icons */
	qproperty-icon: none; 
	
    min-width: 10px;
    max-width: 10px;
    min-height: 10px;
    max-height: 10px;

    border-radius: 5px; 
	/* set background transparent */
    background-color: transparent; 
	padding: 5px;
}

/* style for pre month button ############################################ */

#qt_calendar_prevmonth {
	/* set text for button */
	/*qproperty-text: &amp;quot;&amp;gt;&amp;quot;;*/
	margin-left:5px;
	image: url(:/icons/esq.ico);
}

/* style for next month button ########################################### */
#qt_calendar_nextmonth {
	margin-right:5px;
	image: url(:/icons/dir.ico);
    /* qproperty-text: &amp;quot;&amp;gt;&amp;quot;; */
}
#qt_calendar_prevmonth:hover, 
#qt_calendar_nextmonth:hover {
    background-color: #55aaff;
}

#qt_calendar_prevmonth:pressed, 
#qt_calendar_nextmonth:pressed {
    background-color: rgba(235, 235, 235, 100);
}


/* Style for month and yeat buttons #################################### */

#qt_calendar_yearbutton {
    color: #000;
	margin:5px;
    border-radius: 5px;
	font-size: 13px;
	padding:0px 10px;
}

 #qt_calendar_monthbutton {
	width: 110px;
    color: #000;
	font-size: 13px;
	margin:5px 0px;
    border-radius: 5px;
	padding:0px 2px;
}

#qt_calendar_yearbutton:hover, 
#qt_calendar_monthbutton:hover {
    background-color: #55aaff;
}

#qt_calendar_yearbutton:pressed, 
#qt_calendar_monthbutton:pressed {
    background-color: rgba(235, 235, 235, 100);
}

/* Style for year input lineEdit ######################################*/

#qt_calendar_yearedit {
    min-width: 53px;
    color: #000;
    background: transparent;
	font-size: 13px;
}

/* Style for year change buttons ######################################*/

#qt_calendar_yearedit::up-button { 
	image: url(:/icons/cima.ico);
    subcontrol-position: right;
}

#qt_calendar_yearedit::down-button { 
	image: url(:/icons/baixo.ico);
    subcontrol-position: left; 
}

#qt_calendar_yearedit::down-button, 
#qt_calendar_yearedit::up-button {
	width:10px;
	padding: 0px 5px;
	border-radius:3px;
}

#qt_calendar_yearedit::down-button:hover, 
#qt_calendar_yearedit::up-button:hover {
	background-color: #55aaff;
}

/* Style for month select menu ##################################### */

#calendarWidget QToolButton QMenu {
     background-color: white;

}
#calendarWidget QToolButton QMenu::item {
	/*padding: 10px;*/
}
 #calendarWidget QToolButton QMenu::item:selected:enabled {
    background-color: #55aaff;
}

#calendarWidget QToolButton::menu-indicator {
	/* Remove toolButton arrow */
      /*image: none; */
	nosubcontrol-origin: margin;
	subcontrol-position: right center;
	margin-top: 10px;
	width:20px;
}

/* Style for calendar table ########################################## */
#qt_calendar_calendarview {
	/* Remove the selected dashed box */
    outline: 0px;

	border: 2px solid  #B8E2FF;
	border-top: 0px;
	border-bottom-left-radius: 5px;
	border-bottom-right-radius: 5px;
}

#qt_calendar_calendarview::item:hover {
   border-radius:5px;
	background-color:#aaffff;
}

#qt_calendar_calendarview::item:selected {
    background-color: #55aa7f; 
	border-radius:5px;
}</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>431</width>
     <height>281</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QCalendarWidget QWidget {
	 alternate-background-color: #B8E2FF;
}

/* style for top navigation area ###############################################*/ 

#qt_calendar_navigationbar {
    background-color: #fff;
	border: 4px solid  #B8E2FF;
	border-bottom: 0px;
	border-top-left-radius: 17px;
	border-top-right-radius: 17px;
}

/* style for month change buttons ############################################ */

#qt_calendar_prevmonth, 
#qt_calendar_nextmonth {
	/* border delete */
    border: none;  
    /* delete default icons */
	qproperty-icon: none; 
	
    min-width: 14px;
    max-width: 14px;
    min-height: 14px;
    max-height: 14px;

	
 
	    border-radius: 6px;
	/* set background transparent */
    background-color: transparent; 
	padding: 10px;
}

/* style for pre month button ############################################ */

#qt_calendar_prevmonth {
	/* set text for button */
	/*qproperty-text: &amp;quot;&amp;gt;&amp;quot;;*/
	margin-top: 4px;
	margin-left: 4px;
	image: url(:/icons/esq.ico);
}

/* style for next month button ########################################### */
#qt_calendar_nextmonth {
	margin-top: 4px;
	margin-right: 4px;
	image: url(:/icons/dir.ico);
    /* qproperty-text: &amp;quot;&amp;gt;&amp;quot;; */
}
#qt_calendar_prevmonth:hover{
    background-color: #55aaff;
	border-bottom-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-top-right-radius: 0px;
	border-top-left-radius: 13px
}

#qt_calendar_nextmonth:hover {
    background-color: #55aaff;
	border-bottom-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-top-right-radius: 13px;
	border-top-left-radius: 0px	
}

#qt_calendar_prevmonth:pressed, 
#qt_calendar_nextmonth:pressed {
    background-color: rgba(235, 235, 235, 100);
}


/* Style for month and year buttons #################################### */

#qt_calendar_yearbutton {
    color: #000;
	margin:5px;
    border-radius: 5px;
	margin-top: 9px;
	font-size: 13px;
	padding:0px 10px;
}

 #qt_calendar_monthbutton {
	width: 110px;
    color: #000;
	font-size: 13px;
	margin:5px 0px;
	margin-top: 9px;
    border-radius: 5px;
	padding:0px 2px;
}

#qt_calendar_yearbutton:hover, 
#qt_calendar_monthbutton:hover {
    background-color: #55aaff;
}

#qt_calendar_yearbutton:pressed, 
#qt_calendar_monthbutton:pressed {
    background-color: rgba(235, 235, 235, 100);
}

/* Style for year input lineEdit ######################################*/

#qt_calendar_yearedit {
    min-width: 53px;
    color: #000;
    background: transparent;
	font-size: 13px;
}

/* Style for year change buttons ######################################*/

#qt_calendar_yearedit::up-button { 
	image: url(:/icons/cima.ico);
    subcontrol-position: right;
}

#qt_calendar_yearedit::down-button { 
	image: url(:/icons/baixo.ico);
    subcontrol-position: left; 
}

#qt_calendar_yearedit::down-button, 
#qt_calendar_yearedit::up-button {
	width:10px;
	padding: 0px 5px;
	border-radius:3px;
}

#qt_calendar_yearedit::down-button:hover, 
#qt_calendar_yearedit::up-button:hover {
	background-color: #55aaff;
}

/* Style for month select menu ##################################### */

#calendarWidget QToolButton QMenu {
     background-color: white;

}
#calendarWidget QToolButton QMenu::item {
	/*padding: 10px;*/
}
 #calendarWidget QToolButton QMenu::item:selected:enabled {
    background-color: #55aaff;
}

#calendarWidget QToolButton::menu-indicator {
	/* Remove toolButton arrow */
      /*image: none; */
	nosubcontrol-origin: margin;
	subcontrol-position: right center;
	margin-top: 10px;
	width:20px;
}

/* Style for calendar table ########################################## */
#qt_calendar_calendarview {
	/* Remove the selected dashed box */
    outline: 0px;

	border: 4px solid  #B8E2FF;
	border-top: 0px;
	border-bottom-left-radius: 0px;
	border-bottom-right-radius: 0px;
}

#qt_calendar_calendarview::item:hover {
   border-radius:5px;
	background-color:#aaffff;
}

#qt_calendar_calendarview::item:selected {
    background-color: #55aa7f; 
	border-radius:5px;
}</string>
   </property>
   <property name="gridVisible">
    <bool>false</bool>
   </property>
   <property name="horizontalHeaderFormat">
    <enum>QCalendarWidget::ShortDayNames</enum>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
   <property name="navigationBarVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Ao selecionar uma data, voce pode checar
exatamente em qual dia da escala ela est√°! </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>571</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Segoe UI&quot;;
font: italic 11pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Os dias de folga estao destacados em colorido.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ala.qrc"/>
 </resources>
 <connections/>
</ui>
